File: src/Clients/Web/SamStoreWeb/src/app/models/ShoppingCarts/shopping-cart.DTO.ts
import { ShoppingCartItemDTO } from "./shopping-cart-item.DTO";

export class ShoppingCartDTO {
	public id?: string = "";
	public costumerId?: string = "";
	public total: number = 0;
	public discount: number = 0;
	public totalEstimated: number = 0;
	public items: ShoppingCartItemDTO[] = [];
}

File: src/ApiGateways/SamStore.BFF.Orders/Models/ShoppingCartDTO.cs
namespace SamStore.BFF.Orders.Models
{
    public class ShoppingCartDTO
    {
        public Guid Id { get; set; }
        public Guid CostumerId { get; set; }
        public decimal Total { get; set; }
        public List<ShoppingCartItemDTO> Items { get; set; }
    }
}
